config/paginate.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'page' => 5
```

Comment:
```suggestion
    'page' => 5,
```
convention, sau phần tử cuối của mảng ntn thêm dấu phẩy vào nha

Reply:
Ok bạn